api/policies/authTenant.js

```diff
@@ -76,9 +76,16 @@ module.exports = (req, res, next) => {
          next();
          return;
       }
-      
+
+      if (prefix == "localhost") {
```

Comment:
Practically Speaking: I don't see anything wrong with this.

Theoretically Speaking: It would seem like I am no longer able to test multi tenant installations on my local site using `localhost`.  I dunno ... maybe I want to test out >1 tenant on my `localhost` ... 

Also, to test out the ability to scan a QR code to load a PWA mobile app, I have to put in my IP address instead of `localhost`.  It would seem I can no longer do that.  We might need to open up that ability:

``` 
//// DEV TESTING

// Add in here a queryString check for `tenant`
// so I can use : http://192.168.xx.xx/?tenant=[prefix]
if (ENV[NODE_ENV] == "development" && isNumeric(prefix) && queryString['tenant']) {
   prefix = queryString['tenant'];
}
```

Reply:
`tenantx.localhost` will still work. But I added support for the tenant query param when in development (regardless of host).